scripts/benchmarks/benchmark-loadgen/benchmark-loadgen.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+PARAM_NUM_INSTANCES=${NUM_INSTANCES:-100}
+
+PARAM_COORDINATOR=${COORDINATOR:-127.0.0.1}
+PARAM_COORDINATOR_API_PORT=${COORDINATOR_API_PORT:-7203}
+PARAM_COORDINATOR_DEBUG_PORT=${COORDINATOR_DEBUG_PORT:-7203}
+
+PARAM_DBNODES=${DBNODES:-127.0.0.1}
+PARAM_DBNODE_DEBUG_PORT=${DBNODE_DEBUG_PORT:-9004}
+
+CFG_COORDINATOR="${PARAM_COORDINATOR}:${PARAM_COORDINATOR_API_PORT}"
+
+CFG_SCRAPE_FAKEWEBSERVERS=""
+for N in $(seq 1 $PARAM_NUM_INSTANCES); do
+  PORT=$(expr 8080 + $N - 1)
```

Comment:
Would it be better to pick a more esoteric port range?

Reply:
This is internal to the backend thankfully.